src/common/util/include/openvino/util/file_util.hpp

```diff
@@ -236,34 +227,18 @@ void convert_path_win_style(std::string& path);
 std::string get_ov_lib_path();
 
 #ifdef OPENVINO_ENABLE_UNICODE_PATH_SUPPORT
+std::wstring path_join_w(const std::vector<ov::util::Path>& paths);
+#endif
 
-using FilePath = std::wstring;
-
-inline std::string from_file_path(const FilePath& path) {
-    return wstring_to_string(path);
-}
-
-inline FilePath to_file_path(const std::string& path) {
-    return string_to_wstring(path);
-}
-
-std::wstring get_directory(const std::wstring& path);
-std::wstring path_join_w(const std::vector<std::wstring>& paths);
-
-#else
-
-using FilePath = std::string;
-
-inline std::string from_file_path(const FilePath& path) {
-    return path;
+using FilePath = ov::util::Path::string_type;
+inline std::string from_file_path(const ov::util::Path& path) {
```

Comment:
This function should be removed. Or marked to be removed. The goal is to used `Path` not strings and wstrings.